sycl/test-e2e/ESIMD/unified_memory_api/atomic_update_slm_acc.cpp

```diff
@@ -0,0 +1,26 @@
+//==--- atomic_update_slm_acc.cpp - DPC++ ESIMD on-device test -------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include "Inputs/atomic_update_slm.hpp"
+
+int main(void) {
+  queue q(esimd_test::ESIMDSelector, esimd_test::createExceptionHandler());
+
+  auto dev = q.get_device();
+  std::cout << "Running on " << dev.get_info<info::device::name>() << "\n";
+
+  bool passed = true;
+
+  constexpr bool TestCacheHintProperties = true;
+  passed &= test_main_acc<!TestCacheHintProperties>(q);
```

Comment:
this code could get same fix as for other 3 tests (https://github.com/intel/llvm/commit/[998d1fe](https://github.com/intel/llvm/commit/998d1fe32ea0c189194d7cab1c5bdf883a56976a)32ea0c189194d7cab1c5bdf883a56976a)

Reply:
Done.